### PR TITLE
docs: add 384d vectors to addons table

### PR DESCRIPTION
**Pull request by egor-romanov**
## What kind of change does this PR introduce?

- docs: add 384d vectors to add-ons table

### CODE REVIEW

File: apps/docs/pages/guides/ai/choosing-compute-addon.mdx [hunk 1]
```diff
@@ -21,6 +21,36 @@ The number of dimensions in your embeddings is the most important factor in choo
...
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="gte384"
+>
+<TabPanel id="gte384" label="gte-small-384">
+
+| Plan   | Vectors   | m   | ef_construction | ef_search | QPS  | Latency Mean | Latency p95 | RAM Usage  | RAM    |
+| ------ | --------- | --- | --------------- | --------- | ---- | ------------ | ----------- | ---------- | ------ |
+| Free   | 100,000   | 16  | 64              | 60        | 580  | 0.017 sec    | 0.024 sec   | 1.2 (Swap) | 1 GB   |
```
Comment (kevcodez) [inline apps/docs/pages/guides/ai/choosing-compute-addon.mdx:38]:
```suggestion
| Starter   | 100,000   | 16  | 64              | 60        | 580  | 0.017 sec    | 0.024 sec   | 1.2 (Swap) | 1 GB   |
```

Comment (kevcodez) [reply apps/docs/pages/guides/ai/choosing-compute-addon.mdx:38]: We refer to it as Starter instance, as it's no longer Free with usage-based billing

File: apps/docs/pages/guides/ai/choosing-compute-addon.mdx [hunk 2]
```diff
@@ -33,18 +63,18 @@ This benchmark uses the [dbpedia-entities-openai-1M](https://huggingface.co/data
...
-| XL     | 500,000   | 24  | 56              | 100       | 360  | 0.055 sec    | 0.135 sec   | 13 GB              | 16 GB  |
-| 2XL    | 1,000,000 | 24  | 56              | 250       | 560  | 0.036 sec    | 0.058 sec   | 32 GB              | 32 GB  |
-| 4XL    | 1,000,000 | 24  | 56              | 250       | 950  | 0.021 sec    | 0.033 sec   | 39 GB              | 64 GB  |
-| 8XL    | 1,000,000 | 24  | 56              | 250       | 1650 | 0.016 sec    | 0.023 sec   | 40 GB              | 128 GB |
-| 12XL   | 1,000,000 | 24  | 56              | 250       | 1900 | 0.015 sec    | 0.021 sec   | 38 GB              | 192 GB |
-| 16XL   | 1,000,000 | 24  | 56              | 250       | 2200 | 0.015 sec    | 0.020 sec   | 40 GB              | 256 GB |
+| Plan   | Vectors   | m   | ef_construction | ef_search | QPS  | Latency Mean | Latency p95 | RAM Usage     | RAM    |
+| ------ | --------- | --- | --------------- | --------- | ---- | ------------ | ----------- | ------------- | ------ |
+| Free   | 15,000    | 16  | 40              | 40        | 480  | 0.011 sec    | 0.016 sec   | 1.2 GB (Swap) | 1 GB   |
```
Comment (kevcodez) [inline apps/docs/pages/guides/ai/choosing-compute-addon.mdx:68]:
```suggestion
| Starter   | 15,000    | 16  | 40              | 40        | 480  | 0.011 sec    | 0.016 sec   | 1.2 GB (Swap) | 1 GB   |
```

File: apps/docs/pages/guides/ai/choosing-compute-addon.mdx (hunk 1 above)
Comment (kevcodez) [reply apps/docs/pages/guides/ai/choosing-compute-addon.mdx:38]: You probably have to run prettier again afterwards

File: apps/docs/pages/guides/ai/choosing-compute-addon.mdx (hunk 2 above)
Comment (kevcodez) [reply apps/docs/pages/guides/ai/choosing-compute-addon.mdx:68]: You probably have to run prettier again afterwards

File: apps/docs/pages/guides/ai/choosing-compute-addon.mdx [hunk 3]
```diff
@@ -59,49 +89,61 @@ It is possible to upload more vectors to a single table if Memory allows it (for
...
-| 2XL    | 1,623,397 | 1275  | 1400 | 0.061 sec    | 0.075 sec   | 22 GB              | 32 GB  |
-| 4XL    | 1,623,397 | 1275  | 1800 | 0.027 sec    | 0.043 sec   | 20 GB              | 64 GB  |
-| 8XL    | 1,623,397 | 1275  | 2850 | 0.032 sec    | 0.049 sec   | 21 GB              | 128 GB |
-| 12XL   | 1,623,397 | 1275  | 3700 | 0.020 sec    | 0.036 sec   | 26 GB              | 192 GB |
-| 16XL   | 1,623,397 | 1275  | 3700 | 0.025 sec    | 0.042 sec   | 29 GB              | 256 GB |
+<TabPanel id="gte384_98" label="gte-small-384, accuracy=.98">
+
+| Plan   | Vectors   | Lists | Probes | QPS  | Latency Mean | Latency p95 | RAM Usage     | RAM    |
+| ------ | --------- | ----- | ------ | ---- | ------------ | ----------- | ------------- | ------ |
+| Free   | 100,000   | 500   | 50     | 205  | 0.048 sec    | 0.066 sec   | 1.2 GB (Swap) | 1 GB   |
```
Comment (kevcodez) [inline apps/docs/pages/guides/ai/choosing-compute-addon.mdx:121]: In this file, too, could you rename "Free" to "Starter"?

**Review by kevcodez — APPROVED** [review]

**Review by gregnr — APPROVED** [review]

**Pull request merged**